lib/cmds/update.ts

```diff
@@ -61,12 +61,14 @@ if (argv._[0] === 'update-run') {
   prog.printHelp();
 }
 
+let browserFile: BrowserFile;
 
 /**
  * Parses the options and downloads binaries if they do not exist.
  * @param options
  */
-function update(options: Options): void {
+function update(options: Options): Promise<void> {
+  let promises: any[] = [];
```

Comment:
You can use the more permissive `q.IPromise` type here, since the `Promise` and `q.Promise` types are mutually incompatible